terraform/testnet/addons.tf

```diff
@@ -102,3 +101,44 @@ resource "aws_iam_role_policy" "cluster-autoscaler" {
   role   = aws_iam_role.cluster-autoscaler[0].name
   policy = data.aws_iam_policy_document.cluster-autoscaler[0].json
 }
+
+resource "kubernetes_namespace" "chaos-testing" {
```

Comment:
can we just rename all the places that use `chaos-testing` to `chaos-mesh` instead for consistency and simplicity?

Reply:
I was hoping to be able to re-use the namespace for other things other than `chaos-mesh`, e.g. if we want to look into another chaos testing framework, or if we want to install other resources like Forge helpers

Reply:
I still think it's better to put this into it's own namespace, creating namespaces is cheap. It's a good practice and also makes cleaning things up a lot easier if things go south as you just have to delete the namespace.